c2corg_ui/templates/outing/edit.html

```diff
@@ -111,6 +111,17 @@ <h5 translate ng-show="outing.associations.routes.length == 0">choose at least o
               </div>
             </div>
 
+            ## DURATION DIFFICULTIES
+            <div class="form-group data half" ng-class="{'has-success': outing.duration_difficulties}"
+                 ng-if="outing.activities.indexOf('snow_ice_mixed') > -1 || outing.activities.indexOf('mountain_climbing') > -1 ||
+                           outing.activities.indexOf('rock_climbing') > -1 || outing.activities.indexOf('ice_climbing') > -1 || outing.activities.indexOf('via_ferrata') > -1">
+              <label translate>duration_difficulties</label>
+              <div class="input-group">
+                <input type="number" min="1" minlength="1" name="duration_difficulties" ng-model="outing.duration_difficulties" class="form-control" />
+                <span class="input-group-addon">min</span>
```

Comment:
according to https://docs.google.com/spreadsheets/d/110umbpgS1CUDtzeJ62fNBvn9UBHl_wSbXx5xhFlQTng/edit#gid=792641820  (line 25) - minutes